gtsam/slam/SmartFactorBase.h

```diff
@@ -206,11 +206,12 @@ class SmartFactorBase: public NonlinearFactor {
       boost::optional<Matrix&> E = boost::none) const {
     Vector ue = cameras.reprojectionError(point, measured_, Fs, E);
     if (body_P_sensor_ && Fs) {
-      const Pose3 sensor_P_body = body_P_sensor_->inverse();
+      const Pose3 sensor_T_body = body_P_sensor_->inverse();
```

Comment:
Please keep P, as the field uses that convention

Reply:
Gotcha. I was trying to follow the convention as per Samarth's post. Reverted.